### ISSUE TITLE
Firestore fails to get document with null values

### ISSUE THREAD

**Issue by SimonMayerhofer** [n8n-io/n8n#1747]
**Describe the bug**
I have a document in firestore which looks like this:
```json
{
	"description": null
}
```

When I get this document the workflow fails with the following error:

```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at documentToJson (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Firebase/CloudFirestore/GenericFunctions.js:87:28)
    at documentToJson (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Firebase/CloudFirestore/GenericFunctions.js:105:27)
    at documentToJson (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Firebase/CloudFirestore/GenericFunctions.js:105:27)
    at Object.fullDocumentToJson (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Firebase/CloudFirestore/GenericFunctions.js:82:122)
    at /usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Firebase/CloudFirestore/CloudFirestore.node.js:91:51
    at Array.map (<anonymous>)
    at Object.execute (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Firebase/CloudFirestore/CloudFirestore.node.js:90:68)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/WorkflowExecute.js:420:47
```

**To Reproduce**
1. Add document to Firestore with null value
2. Get the document with the Firestore node
3. See error

**Expected behavior**
Document is returned without failing.

**Environment (please complete the following information):**
 - OS: Ubuntu Linux 20.04
 - n8n Version 0.118.1

**Comment by janober** [comment]
Issue got fixed by @ivov with #1749 which just got merged and will be released with next version.

**Comment by janober** [comment]
Got released with n8n@0.119.0
